### ISSUE TITLE
Explorer font exception

### ISSUE THREAD

**Issue by a562314** [Rectify11/Installer#621]
### Description

When I switch to a light theme, the Explorer font breaks up badly, switch to a dark theme and it goes back to normal.
![屏幕截图 2024-09-08 203712](https://github.com/user-attachments/assets/5b98a4c8-f2be-4458-aaa2-0511caff945f)
![屏幕截图 2024-09-08 203832](https://github.com/user-attachments/assets/4ff8215a-6faa-4e3f-a14c-1bbcd2d8007a)

Note that the dark theme I used with the Rectify 11 install down, the fonts were normal when I switched between light and dark colors in Windows 11 settings, once I switched to Rectify 11's light theme, the fonts were still broken no matter how I switched them in Windows settings.

### Steps To Reproduce

When installing the light color theme that comes with Rectify 11, opening Explorer fonts are broken and distorted

### Rectify11 Version

3.2.70.2

### Windows Version

Windows 11 24H2 26100.1591

### Logs

![屏幕截图 2024-09-08 203712](https://github.com/user-attachments/assets/b578773c-1809-4e6f-8c43-ef4d3836b032)
![屏幕截图 2024-09-08 203832](https://github.com/user-attachments/assets/738c86e3-b306-46ed-a853-71bff739a838)

**Comment by a562314** [comment]
═════════════════════════════
START: 2024/9/8 16:41:52
Rectify11 Version: 3.2.70.2
═════════════════════════════
Preparing Installation
──────────────────────
Wrote RectifyStart.exe
Wrote 7za.exe
Wrote files.7z
Wrote extras.7z
Wrote ResourceHacker.exe
WriteFiles() succeeded.
Extracting vcredist.exe from extras.7z
Extracting core31.exe from extras.7z
Executing vcredist.exe with arguments /install /quiet /norestart
vcredist.exe exited with error code 1638
Executing core31.exe with arguments /install /quiet /norestart
core31.exe exited with error code 0
InstallRuntimes() succeeded.
══════════════════════════════════════════════
Wrote files.7z
Extracted files.7z
Wrote ThemeDLL.dll
Installr11cpl() succeeded.
Installing Extras
─────────────────
UninstallGadgets() succeeded.
InstallGadgets() succeeded.
UninstallShell() succeeded.
Setting menu type: NilesoftShellAll
InstallShell() succeeded.
UninstallUserAv() succeeded.
InstallUserAvatars() succeeded.
Extras.Install() succeeded.
══════════════════════════════════════════════
Installing icons
────────────────
MatchAndApplyRule() succeeded.
Wrote to PendingFiles
Wrote to x86PendingFiles
Wrote to Language
Wrote to Version
Wrote to WindowsUpdate
Wrote to OSVersion
WritePendingFiles() succeeded.
Wrote RectifyStart.exe
Wrote NSudoL.exe
Wrote Rectify11.Phase2.exe
Wrote Rectify11.Phase2.dll
Wrote Rectify11.Phase2.runtimeconfig.json
WriteFiles() succeeded.
screensaver.reg succeeded.
MmcHelper.PatchAll() succeeded.
FixOdbc() succeeded.
icons.reg succeeded.
Icons.Install() succeeded.
══════════════════════════════════════════════
Installer copied to C:\WINDOWS\Rectify11\Uninstall.exe
══════════════════════════════════════════════
Cleaning up
───────────
Cleanup() succeeded.
══════════════════════════════════════════════

**Comment by mina354** [comment]
Yes, it's caused by ExplorerFrame.exe, I have to end that process every time just to return font display to normal. I reported it here: #592

"There's also this process called ExplorerFrame that I have to end every time, it makes the text on file details in File Explorer difficult to read."
